<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EW-Builder</string>
  </property>
  <property name="windowIcon">
   <iconset theme="system-run">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="lineLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Line layer</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelPattern">
      <property name="text">
       <string>Label pattern</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>582</width>
         <height>198</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="valuesLayout"/>
      </widget>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="Point"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="classLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Class</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="Pattern"/>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="Text"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="Class"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="Line"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="pointLabel">
      <property name="text">
       <string>Point layer</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="textLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Text layer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainTool">
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionProceed"/>
   <addaction name="actionJobs"/>
   <addaction name="actionGeometry"/>
   <addaction name="actionDuplicates"/>
   <addaction name="actionInvisivle"/>
   <addaction name="actionLabels"/>
   <addaction name="separator"/>
   <addaction name="actionHide"/>
  </widget>
  <widget class="QToolBar" name="supportTool">
   <property name="windowTitle">
    <string>Support</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCancel"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset theme="network-wired">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="icon">
    <iconset theme="network-offline">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionProceed">
   <property name="icon">
    <iconset theme="system-run">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Proceed</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="icon">
    <iconset theme="stop">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionJobs">
   <property name="icon">
    <iconset theme="view-refresh">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Update jobs</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionGeometry">
   <property name="icon">
    <iconset theme="view-fullscreen">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Fit geometry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="tools-check-spelling">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Hide unused layers</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="actionInvisivle">
   <property name="icon">
    <iconset theme="edit-cut">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Hide intersections</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionDuplicates">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Remove duplicates</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLabels">
   <property name="icon">
    <iconset theme="format-text-italic"/>
   </property>
   <property name="text">
    <string>Fit labels</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Class</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>classIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>668</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>classIndexChanged(int)</slot>
 </slots>
</ui>
